Core/Core/Contexts/Context.swift

```diff
@@ -95,3 +96,22 @@ public extension Context {
         return context.id.localID == id.localID
     }
 }
+
+public extension Context {
+
+    var color: UIColor? { contextColor?.color }
+
+    var contextColor: ContextColor? {
+        contextColor(in: AppEnvironment.shared.database.viewContext)
+    }
```

Comment:
I would remove these helpers to make it explicit that these properties are not part of the `Context` object and fast to access because in the background they do a CoreData fetch that could have negative effect on performance if used frequently for example during SwiftUI rendering. The functions below are clearer for me that they work with CoreData to produce the result and might be slow.